tracker-neuralnet/ftnoir_tracker_neuralnet.cpp

```diff
@@ -900,6 +928,31 @@ void NeuralNetDialog::startstop_trans_calib(bool start)
 }
 
 
+void NeuralNetDialog::onSelectPoseNetFile()
+{
+    const auto root = get_default_model_directory();
+    // Start with the current setting
+    QString filename = settings_.posenet_file;
+    // If the filename is relative then assume that the file is located under the
+    // model directory. Under regular use this should always be the case.
+    if (QFileInfo(filename).isRelative())
+        filename = root.absoluteFilePath(filename);
+    filename = QFileDialog::getOpenFileName(this,
+        tr("Select Pose Net ONNX"), filename, tr("ONNX Files (*.onnx)"));
+    // In case the user aborted.
+    if (filename.isEmpty())
+        return;
+    // When a file under the model directory was selected we can get rid of the
+    // directory prefix. This is more robust than storing absolute paths, e.g.
+    // in case the user moves the opentrack install folder / reuses old settings.
+    // When the file is not in the model directory, we have to use the absolute path,
+    // which is also fine as developer feature.
+    if (filename.startsWith(root.absolutePath()))
+        filename = root.relativeFilePath(filename);
+    settings_.posenet_file = filename;
```

Comment:
It's going to crash and burn with no user-facing indication when the file can't be opened for reading, right?

Reply:
:sweat_smile: It was intended not to. But an exception got through.

Now tested with invalid file (the localizer onnx) and non-existing file. There will be no explicit error indication. The tracker just won't start. Ok?

Reply:
It's fine as-is. FWIW, you can out `run_in_thread_async` or `run_in_thread_sync`. They're a bit evil but in a good way.

Reply:
...Or return `module_status` instead of `bool`.